common/static/common/templates/discussion/thread-type.underscore

```diff
@@ -1,15 +1,14 @@
 <div class="post-field">
-    <label class="field-label">
-        <span class="field-label-text">
+    <fieldset class="field-label">
+        <label class="field-label-text">
```

Comment:
does this need a `for` attribute like the file below?

Reply:
I think it might not need it because it's within a fieldset, but I was just deferring to Mark's example of what was acceptable.